tests/integration_tests/test_ray.py

```diff
@@ -1015,15 +1014,14 @@ def create_dataset_pipeline(
             "output_features": [{"name": "out_column", "type": "binary"}],
             TRAINER: {"epochs": 1, BATCH_SIZE: 128},
         }
-        backend_config = {**RAY_BACKEND_CONFIG}
+        backend_config = copy.deepcopy(RAY_BACKEND_CONFIG)
+        backend_config["loader"] = {"window_size_bytes": window_size_bytes}
```

Comment:
Just curious - are we always guaranteed to have the `loader` key in the backend config? If not, do we need to add a default value for this in the schema (possibly `None`)?

Reply:
Under the hood, this update uses `backend._data_loader_kwargs` which the `RayBackend` initializer sets with `self._data_loader_kwargs = loader or {}`. We don't actually rely on the existence of `loader`, but rather the backend's handling of its config.